config/linkcheck-suppressions.txt

```diff
@@ -1333,6 +1335,7 @@
 <a href="com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.html#%3Cinit%3E(com.puppycrawl.tools.checkstyle.TreeWalkerAuditEvent,int)">com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.html#%3Cinit%3E(com.puppycrawl.tools.checkstyle.TreeWalkerAuditEvent,int)</a>: doesn't exist.
 <a href="com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.html#%3Cinit%3E(com.puppycrawl.tools.checkstyle.api.DetailAST,int,int,com.puppycrawl.tools.checkstyle.api.FileText,int)">com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.html#%3Cinit%3E(com.puppycrawl.tools.checkstyle.api.DetailAST,int,int,com.puppycrawl.tools.checkstyle.api.FileText,int)</a>: doesn't exist.
 <a href="com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.html#%3Cinit%3E(com.puppycrawl.tools.checkstyle.api.DetailAST,int,int,int,com.puppycrawl.tools.checkstyle.api.FileText,int)">com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.html#%3Cinit%3E(com.puppycrawl.tools.checkstyle.api.DetailAST,int,int,int,com.puppycrawl.tools.checkstyle.api.FileText,int)</a>: doesn't exist.
+<a href="config_blocks.html#LeftCurly">config_blocks.html#LeftCurly</a>: doesn't exist.
```

Comment:
I would need proof on why this is showing up as not existing. It looks like a valid link.

Reply:
I'm not so sure about this, the CI was complaining to add this line so I just followed its instructions. Maybe this could be because of change you mentioned here? https://github.com/checkstyle/checkstyle/pull/14643#discussion_r1583429191 

I guess I accidently updated that line. I will fix it and remove this suppression and see if the CI is still complaining or not.

Reply:
@rnveach I revert the changes from here and from https://github.com/checkstyle/checkstyle/pull/14643#discussion_r1583429191 but it is still giving the same error: https://app.circleci.com/pipelines/github/checkstyle/checkstyle/25403/workflows/21f8a1a1-a150-477d-9704-5aa654f78ed2/jobs/584953?invite=true#step-103-245950_98

I also don't know why this is happening.